src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/CompiledRegexRunner.cs

```diff
@@ -1,17 +1,22 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Globalization;
+
 namespace System.Text.RegularExpressions
 {
     internal sealed class CompiledRegexRunner : RegexRunner
     {
         private readonly ScanDelegate _scanMethod;
+        /// <summary>This field will only be set if the pattern contains backreferences and has RegexOptions.IgnoreCase</summary>
+        private readonly TextInfo? _textInfo;
```

Comment:
Where is this used?

Reply:
This gets accessed via reflection here:

https://github.com/dotnet/runtime/blob/6b4f89362280dcb019071b2e277ce7d306f3ea5c/src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCompiler.cs#L24

and is later used to call ToLower when evaluating a Backreference with IgnoreCase:

https://github.com/dotnet/runtime/blob/6b4f89362280dcb019071b2e277ce7d306f3ea5c/src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCompiler.cs#L340-L342